core/src/Revolution/Processors/Security/ResourceGroup/GetNodes.php

```diff
@@ -85,13 +85,13 @@ public function process()
                     $resources = $resourceGroup->getResources();
                     /** @var modResource $resource */
                     foreach ($resources as $resource) {
+                        $icons = $resource->getIconClasses();
```

Comment:
If electing to add statuses:
```suggestion
                        $icons = $resource->getIconClasses();
                        $cls = [];
                        if (!$resource->get('published')) {
                            $cls[] = 'unpublished';
                        }
                        if ($resource->get('deleted')) {
                            $cls[] = 'deleted';
                        }
                        if ($resource->get('hidemenu')) {
                            $cls[] = 'hidemenu';
                        }
```

Reply:
Would be better to put this logic in the `getIconClasses()` method, no?

Reply:
Actually now that I think about it longer than 5 seconds, no, because `cls` is applied somewhere else than `iconCls`. So support for this suggestion.